Source/Core/DolphinQt/Config/LogConfigWidget.cpp

```diff
@@ -47,6 +47,8 @@ void LogConfigWidget::CreateWidgets()
   m_verbosity_error = new QRadioButton(tr("Error"));
   m_verbosity_warning = new QRadioButton(tr("Warning"));
   m_verbosity_info = new QRadioButton(tr("Info"));
+  m_verbosity_debug = new QRadioButton(tr("Debug"));
+  m_verbosity_debug->setDisabled(MAX_LOGLEVEL < 5);
```

Comment:
Whatever this ends up being in the end, could you please use `LDEBUG` instead of `5` there?